src/backend/api/meals.js

```diff
@@ -2,13 +2,28 @@ const express = require("express");
 const router = express.Router();
 const knex = require("../database");
 
-router.get("/", async (request, response) => {
-  try {
+//router.get("/", async (request, response) => {
+  //try {
     // knex syntax for selecting things. Look up the documentation for knex for further info
-    const titles = await knex("meals").select("title");
-    response.json(titles);
-  } catch (error) {
-    throw error;
+   // const titles = await knex("meals").select("title");
+    //response.json(titles);
+  //} catch (error) {
+    //throw error;
+  //}
+//});
+
+router.get("/", async (request, response) => {
+  try{
+const mealWithDescription = await knex  ("meal")
+.leftJoin('Review', 'meal.id', 'Review.meal_id')
+.select("meal.title","meal.location","meal.price","Review.description")
+
+response.json(mealWithDescription);
+
+
+  }catch(error){
+    console.log("error", error)
```

Comment:
In case of an error here, we are not returning any HTTP response. So the client will be waiting for the response forever. For example the way you have done it in your reservations.js is a good one.